upgrade-notes/backend_20240726_093001.md

```diff
@@ -0,0 +1,53 @@
+#### product files support ([#{3288}](https://github.com/shopsys/shopsys/pull/3288))
+
+-   uploaded files changes
+
+    -   translations introduced for `UploadedFile` entity for support of localized names
+        -   `UploadedFileTranslation` entity was introduced
+        -   `UploadedFile` now extends `AbstractTranslatableEntity`
+        -   `UploadedFileData` properties added
+        -   `UploadedFileFacade::manageFiles` method was updated to handle translations
+        -   `UploadedFileFacade::uploadFile` and `UploadedFileFacade::uploadFiles` methods updated and added a required parameter for names
+        -   `MultiLocaleFileUploadType` was introduced to handle file uploads with translations
+    -   relations support
+        -   `UploadedFile` entity fields `entityName`, `entityId`, `type` and `position` were moved to `UploadedFileRelation`, enabling the storage of multiple relations for one file. Data needs to be migrated over.
+        -   `UploadedFile` constructor was updated to reflect the new structure
```

Comment:
in the upgrade notes, we are trying to explicitly show how the interfaces changed using diffs, see e.g. https://github.com/shopsys/shopsys/blob/15.0/upgrade-notes/backend_20240529_084428.md#add-rounding-and-discount-as-a-separate-order-item-types-3056

I know it can be exhausting so maybe we can discuss this kind of upgrade instructions within the team but I am afraid this general information has not that big added value for those who are actually doing the upgrade of their project

Reply:
upgrade notes rewritten